anu-001/FirstForFood
# #160871718 Refactor Tests

#### What does this PR do?
Provide more tests to all controllers
#### Description of Task to be completed?
Testing Driven Development,
#### How should this be manually tested?
Clone the project repository,  and run; npm run test, to see the testing passing
#### What are the relevant pivotal tracker stories?
[#160871718](https://www.pivotaltracker.com/story/show/160871718)
